pkg/context/context.go

```diff
@@ -265,13 +266,19 @@ func (c *Context) WriteMixinOutputToFile(filename string, bytes []byte) error {
 
 // SetSensitiveValues sets the sensitive values needing masking on output/err streams
 func (c *Context) SetSensitiveValues(vals []string) {
-	if len(vals) > 0 {
+	if len(vals) > 0 && !c.ShowSensitiveValues {
```

Comment:
This function is used to update the censored writer with new values to *** out of the logs, when they are written later. When ShowSensitiveValues is true, we can preemptively exit the function without further changes.

The changes here will immediately print the sensitive values from the disallow list to the console, which isn't appropriate for this function.